README.md

```diff
@@ -59,13 +59,15 @@ Alternatively, you can configure username and password in the `sonatype` block:
 nexusPublishing {
     repositories {
         sonatype {
-            username = "your-username"
-            password = "your-password"
+            username = "your-username"    //preferably your API username
+            password = "your-password"    //preferably your API key
         }
     }
 }
 ```
 
+To increase security, it is advides to use the [API username and key pair](https://blog.solidsoft.pl/2015/09/08/deploy-to-maven-central-using-api-key-aka-auth-token/) (instead of regular credentials).
```

Comment:
```suggestion
To increase security, it is advised to use the [API username and key pair](https://blog.solidsoft.pl/2015/09/08/deploy-to-maven-central-using-api-key-aka-auth-token/) (instead of regular credentials).
```

Small typo here. Thanks for the blog post link!

Reply:
Fixed, thanks.